examples/run_tests_prom.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+function assert_prom_no_response() {
+  curl --silent --fail "$1" >/dev/null
+  status="$?"
+  if [ "$status" -eq 0 ]; then
+    echo FAIL
+    exit 1
+  else
+    echo OK
+  fi
+}
+
+function assert_prom_response() {
+  # Match something that looks like the expected body
+  curl --silent --fail "$1" | grep 'TYPE\|HELP\|counter\|gauge' >/dev/null
```

Comment:
Please use wget instead of curl - it is not installed

Reply:
@eakraly is there anything else needed from our side?